docker-compose.yml

```diff
@@ -65,7 +65,7 @@ services:
       - local
   scheduler:
     image: fnndsc/cube:next-3.0.0.a2.git.ba5ff6f
-    command: celery -A core beat -l info --pidfile /home/localuser/celerybeat.pid --scheduler django_celery_beat.schedulers:DatabaseScheduler
+    command: bash -c "rm -f /home/localuser/celerybeat.pid && celery -A core beat -l info --pidfile= --scheduler django_celery_beat.schedulers:DatabaseScheduler"
```

Comment:
don't need `bash -c "rm -f ...` nor `pidfile= `

Reply:
Sorry , I forgot to delete the rm command ..  look at the latest commit I have fixed it.. now..